lib/Bundler.js

```diff
@@ -48,7 +48,7 @@ export class Bundler {
   async runBundler() {
     const { logger } = this;
     logger.info('Bundling API Console...');
-    let command = './node_modules/.bin/rollup';
+    let command = '.\\node_modules\\.bin\\rollup';
```

Comment:
This will break MacOS and Linux usage. It should use `path.join()` to build the path.